log_drains/list_addon.go

```diff
@@ -0,0 +1,83 @@
+package log_drains
+
+import (
+	"os"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/go-scalingo"
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+)
+
+type addonObject struct {
+	Drains    []scalingo.LogDrain
+	AddonName string
+}
+
+func ListAddon(app string, addonID string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	resources, err := c.AddonsList(app)
```

Comment:
```suggestion
	addons, err := c.AddonsList(app)
```

`resources` does not look like a good variable name, don't you think?